internal/saucecloud/imagerunner.go

```diff
@@ -68,6 +69,11 @@ func (r *ImgRunner) RunProject() (int, error) {
 		return 1, err
 	}
 
+	if r.Project.DryRun {
+		log.Info().Msgf("The following test suites would have run: [%s].", r.getSuiteNames())
```

Comment:
I always found that style of output hard to read, especially when you have
1) lots of suites
2) long suite names

```
The following test suites would have run: [suite-0, suite-1, suite-2, suite-3, suite-4, some suite with a long descriptive name, suite-6, suite-7, suite-8, suite-9]
```

I think it's easier on the eyes if this is using a more vertical printout instead:
```
The following test suites would have run:
  - suite-0
  - suite-1
  - suite-2
  - ...
```

What you think?

Reply:
I agree. I mainly kept that output to be consistent with others.
I'll update all of them to gain clarity.